jax/api.py

```diff
@@ -565,21 +565,28 @@ def vmap(fun, in_axes=0, out_axes=0):
 
   Args:
     fun: Function to be mapped over additional axes.
-    in_axes: Specifies which input axes to map over. Normally this is a tuple with
-       one axes specification for each function argument. An integer is interpreted
-       as a tuple with the same value for all arguments. One argument axes specification
-       can be an integer (0 means first dimension), None (means that the dimension is
-       broadcasted). If the argument is a tuple of values, then the axes specification
-       can be a matching tuple as well.
-
-    out_axes: Specifies which output axes to map over. These may be integers,
-      `None`, or (possibly nested) tuples of integers or `None`.
+    in_axes: A nonnegative integer, None, or (nested) standard Python container
+      (tuple/list/dict) thereof specifying which input array axes to map over.
+      If each positional argument to ``fun`` is an array, then ``in_axes`` can
+      be a nonnegative integer, a None, or a tuple of integers and Nones with
+      length equal to the number of positional arguments to ``fun``. An integer
+      or None indicates which array axis to map over for all arguments (with
+      None indicating not to map any axis), and a tuple indicates which axis to
+      map for each corresponding positional argument. More generally, if the
+      positinal arguments to ``fun`` are container types, the corresponding
```

Comment:
```suggestion
      positional arguments to ``fun`` are container types, the corresponding
```

Reply:
That was amazing. I haven't figured out how to do this on github _or_ in google.